src/pages/filter/ui/custom-inheritance-mode.tsx

```diff
@@ -124,6 +129,16 @@ export const CustomInheritanceMode = observer(
 
       const params = `{"scenario":{${scenarioString}}}`
 
+      filterStore.setFilterCondition<ICustomInheritanceFormValues>(
+        FuncStepTypesEnum.CustomInheritanceMode,
+        {
+          first: firstSelectValue,
+          second: secondSelectValue,
+          third: thirdSelectValue,
+          reset: resetValue,
+          scenario: scenarioString,
```

Comment:
create a state for scenario similar to `***Value` things, and set the scenario here. Therefore, you won't need to duplicate the `setFilterCondition` call since it will be triggered automatically.